src/spikeinterface/core/node_pipeline.py

```diff
@@ -129,7 +136,7 @@ def compute(self, traces, start_frame, end_frame, segment_index, max_margin):
 
         # make sample index local to traces
         local_peaks = local_peaks.copy()
-        local_peaks["sample_index"] -= start_frame - max_margin
+        local_peaks["sample_index"] -= start_frame - left_margin
```

Comment:
@samuelgarcia this was wrong for the first chunk!!!

Reply:
no because of the zero padding